frontend/packages/console-dynamic-plugin-sdk/src/coderefs/coderef-utils.ts

```diff
@@ -1,7 +1,8 @@
 /* eslint-disable no-console */
 
 import { CodeRef } from '../types';
-import { codeRefSymbol } from './coderef-resolver';
+
+export { applyCodeRefSymbol as getExecutableCodeRef } from './coderef-resolver';
```

Comment:
cc @rohitkrai03 @christianvogt

The `applyCodeRefSymbol` function replaces the existing `getExecutableCodeRef` helper.